fluentbit/dockerdir/opt/percona/logrotate/logrotate-mysql.conf

```diff
@@ -0,0 +1,14 @@
+/var/lib/mysql/*.log {
+   create 660 mysql mysql
+   daily
+   minsize 10M
+   maxsize 100M
+   rotate 10
+   missingok
+   nocompress
```

Comment:
why not compress?

Reply:
what do you think?

Reply:
@delgod It is good question because if we want to compress the logs we need to add several additional dependencies into docker image and so on...

Reply:
lets ask SergeyP

Reply:
from SergeyP, Let's leave uncompressed.